serverless-chatbot/README.md

```diff
@@ -0,0 +1,15 @@
+# Serverless Chatbot
+
+This is an example project that shows how to create a serverless chatbot that interacts with Beekeeper users and is triggered by `CHATS.MESSAGE.CREATED` webhook event.
+
+## Prerequisites 
+1. Make sure you have Node.js installed on your machine.
+2. Follow [the instructions](https://serverless.com/framework/docs/providers/aws/guide/installation/) to install the Serverless framework.
+3. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [configure the credentials](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+## Deploying
+1. Copy `config.yml.sample` file as `config.yml` and fill in Beekeeper tenant URL and API token.
```

Comment:
This API token would rather be a bot one, most probably the one that created a webhook. It'd be good to describe this (as mentioned above, so that you understand how this all works together)

Reply:
Done, linked the instruction for creating the bot account.